engine/src/utils.cc

```diff
@@ -84,10 +84,9 @@ int my_system_r(nagios_macros* mac,
   // time to start command.
   gettimeofday(&start_time, nullptr);
 
-  std::shared_ptr<commands::raw_v2> raw_cmd =
-      std::make_shared<commands::raw_v2>(g_io_context, "system", cmd);
+   commands::raw raw_cmd("system", cmd);
```

Comment:
we're missing this bit compared to the other branches:
```
  // send event broker.
  broker_system_command(NEBTYPE_SYSTEM_COMMAND_START, NEBFLAG_NONE,
                        NEBATTR_NONE, start_time, end_time, *exectime, timeout,
                        *early_timeout, service::state_ok,
                        const_cast<char*>(cmd.c_str()), nullptr, nullptr);
```
(see https://github.com/centreon/centreon-collect/pull/2643/files#diff-1b72015e48d5a6bd69c4280dbee87e490bf5f130f3ea90ab1285f623b82e3aceR86)
@sechkem do you know the impact or if it's intentional?